spider_robot:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  leg11_leg111_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg11_leg111
    pid: {p: 100.0, i: 1.0, d: 10.0}

  leg22_leg222_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg22_leg222
    pid: {p: 100.0, i: 1.0, d: 10.0}

  leg33_leg333_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg33_leg333
    pid: {p: 100.0, i: 1.0, d: 10.0}

  leg44_leg444_position_controller:
    type: effort_controllers/JointPositionController
    joint: leg44_leg444
    pid: {p: 100.0, i: 1.0, d: 10.0}





